### DISCUSSION TITLE
⏳ ⛔ Loop Carousel With Off Canvas Support (Quick View)

### DISCUSSION THREAD

**Discussion by BlackStar-95** [elementor/elementor#29631] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Adding support for the new Off canvas inside the loop

### Describe the solution you'd like.

It will be great if elementor add support for the off canvas in Loop Carousel 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by Mesmer99** [comment]
Agree

**Comment by Kuulari** [comment]
Hello, great feature. Would be great if this could be prioritised

**Comment by SlowMagic22** [comment]
Would be super useful!

**Comment by Nofratatu** [comment]
Very useful!

**Comment by ryvikl** [comment]
yes please

**Comment by giannisgx89** [comment]
Waiting for this to be added

**Comment by sofardy** [comment]
I am waiting
